### PR TITLE
Add getFrogbot.sh remote details in docs

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/frogbot#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
---

### CODE REVIEW

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
I'd be happy to review this PR again following the implementation of my suggestions.

File: docs/install-gitlab.md [hunk 1]
```diff
@@ -141,6 +138,10 @@ frogbot-scan:
     # For Linux / MacOS runner:
     - curl -fLg "https://releases.jfrog.io/artifactory/frogbot/v2/[RELEASE]/getFrogbot.sh" | sh
     - ./frogbot ${FROGBOT_CMD}
+      
+    # For Linux / MacOS runner using Artifactory remote repository:
+    # - $JF_URL/artifactory/$JF_RELEASES_REPO/artifactory/frogbot/v2/[RELEASE]/getFrogbot.sh" | sh
+    # - ./frogbot ${FROGBOT_CMD}
```
Comment (eyalbe4) [inline docs/install-gitlab.md:144]: Shouldn't we implement the "if statement" here as well?
Can we also support Windows here?

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
I'd be happy to review this again following the implementation of my suggestions.

File: docs/install-azure-repos.md [hunk 2]
```diff
@@ -158,12 +170,13 @@ To install Frogbot on Azure Repos repositories, follow these steps.
...
+         displayName: 'Download and Run Frogbot'   
          inputs:
            script: |
-             curl -fLg "https://releases.jfrog.io/artifactory/frogbot/v2/[RELEASE]/getFrogbot.sh" | sh
-             ./frogbot scan-pull-requests
-             ./frogbot scan-and-fix-repos
+              getFrogbotScriptPath=$( [[ -z "$JF_RELEASES_REPO" ]] && echo "https://releases.jfrog.io" || echo "${JF_URL}/artifactory/${JF_RELEASES_REPO}" )
+              curl -fLg "$getFrogbotScriptPath/artifactory/frogbot/v2/[RELEASE]/getFrogbot.sh" | sh
+              ./frogbot scan-pull-requests
+              ./frogbot scan-and-fix-repos
```
[the remark this answers is not in the document]
Comment (omerzi) [reply docs/install-azure-repos.md:179]: The `task` runs on Linux.

File: docs/install-gitlab.md (hunk 1 above)
Comment (omerzi) [reply docs/install-gitlab.md]: done

**Pull request merged**